packages/lib/src/components/Dropin/Dropin.tsx

```diff
@@ -184,6 +185,10 @@ class DropinElement extends UIElement<DropinConfiguration> implements IDropin {
         this.dropinRef.closeActivePaymentMethod();
     }
 
+    protected onEnterKeyPressed(obj: OnKeyPressObj) {
+        super.onEnterKeyPressed(obj, this.activePaymentMethod || this);
+    }
```

Comment:
I think `super.onEnterKeyPress` might not be needed here. You could call the `this.activePaymentMethod.onEnterKeyPress` directly

Reply:
yup, good point